docs/pages/build/setup.md

```diff
@@ -3,17 +3,16 @@ title: Creating your first build
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight'
-import { InlineCode } from '~/components/base/code';
 
 In this guide, you'll learn how to build a ready-to-submit binary for the Apple App Store and Google Play Store using EAS Build.
 
-Alternatively, if you prefer to sideload the app to your Android device / emulator or install it in the iOS simulator, we'll point you towards resources that explain how to do that.
+Alternatively, if you prefer to install the app directly to your Android device / emulator or install it in the iOS simulator, we'll point you towards resources that explain how to do that.
 
-For a simple app, you should expect to have kicked off your builds for Android and iOS within a few minutes.
+For a small app, you should expect to have kicked off your builds for Android and iOS within a few minutes. If you encounter any issues along the way, you can reach out on the [Expo forums](https://forums.expo.dev/) or [Discord](https://chat.expo.dev/).
```

Comment:
May want to remove Discord from suggestions to set expectations more accurately, overall team attention is mostly on forums esp. the EAS Professionals board